pay-api/src/pay_api/services/internal_pay_service.py

```diff
@@ -0,0 +1,69 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Service to manage Internal Payments.
+
+There are conditions where the payment will be handled internally. For e.g, zero $ or staff payments.
+"""
+
+from datetime import datetime
+from typing import Any, Dict, Tuple
+
+from flask import current_app
+
+from pay_api.services.base_payment_system import PaymentSystemService
+from pay_api.services.invoice import Invoice
+from pay_api.services.payment_account import PaymentAccount
+from pay_api.utils.enums import PaymentSystem
+
+from .oauth_service import OAuthService
+from .payment_line_item import PaymentLineItem
+
+
+class InternalPayService(PaymentSystemService, OAuthService):
+    """Service to manage internal payment."""
+
+    def get_payment_system_url(self, invoice: Invoice, return_url: str):
+        """Return the payment system url."""
+        return None
+
+    def get_payment_system_code(self):
+        """Return INTERNAL as the system code."""
+        return PaymentSystem.INTERNAL.value
+
+    def create_account(self, name: str, account_info: Dict[str, Any]):
+        """Create account internal."""
```

Comment:
Is this implementation complete or is this just a placeholder?

Reply:
Currently we don't create internal accounts, but this implementation is needed to satisfy the abstract super class.